### DISCUSSION TITLE
How to move the image horizontal direction?

### DISCUSSION THREAD

**Discussion by archwolf118** [wkentaro/labelme#1067] [Q&A / Help]
If there is a hot key (move the image from left to right) can get the feature when I was annotating the polygon.
The scroll of mouse can achieve the up and down direction of image, but if I need the move the image from left to right, I have to drag the bottom bar.
If there is a hotkey to drag the whole image or to move the image horizontal?
Thank you!

**Comment by rkutty1** [comment]
@archwolf118  Alt + scroll wheel should be able to move the image from left to right.

**Reply by Alan19922015** [reply]
nice
